nemo_text_processing/text_normalization/data/sentence_boundary_exceptions.txt

```diff
@@ -1,15 +1,4 @@
-mr.
```

Comment:
Do we need this file or can we use the whitelist? added to our to-do

Reply:
thanks, yes its todo